phc/easy/frame.py

```diff
@@ -134,7 +134,8 @@ def expand(
                 # produce local datetime. The column name will have ".local" as
                 # suffix so it'll be clear what's happening.
                 localized = pd.to_datetime(
-                    combined[column_key].str.replace(TZ_REGEX, ""), utc=True
+                    combined[column_key].str.replace(TZ_REGEX, "", regex=True),
```

Comment:
This was the core issue. I want ahead and add another generic the exception to be a catch all anyway so that we're more covered.